site/docs/v1/tech/apis/api-explorer.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: doc
+title: API Explorer
+description: Explore the API for FusionAuth
+---
+
+++++
+<link rel="stylesheet" type="text/css" href="./swagger-ui.css" />
+<style>
+
+/* overrides docs css that causes code text to be dark */
+code.language-json  p {
+color: #FFFFFF;
+}
+code.language-json  span {
+color: #FFFFFF;
+}
+code.language-bash  p {
+color: #FFFFFF;
+}
+code.language-bash  span {
+color: #FFFFFF;
+}
+
+pre.microlight span.headerline {
+color: #FFFFFF;
+}
+
+/* hides smartbear image */
+.topbar-wrapper a.link {
+display:none;
+}
+</style>
+++++
+
+== Overview
+
+This tool uses the https://github.com/fusionauth/fusionauth-openapi[FusionAuth OpenAPI specification] to let you explore APIs FusionAuth offers.
+
+* <<Configuration>>
+** <<In-Browser API Calls>>
+* <<Explorer>>
+
+== Configuration
+
+This tool defaults to using the latest version of the FusionAuth API. If you are using a different version, find the https://github.com/FusionAuth/fusionauth-openapi/tags[correct version] and update the URL in the input box, then click [field]#Explore#.
+
+For example, to view the 1.40.0 release, put `https://raw.githubusercontent.com/FusionAuth/fusionauth-openapi/1.40.0/openapi.yaml` in the YAML file input box.
+
+Don't forget to click on the [field]#Authorize# button to add your FusionAuth API key.
+
+=== In-Browser API Calls
+
+If you want to use the in-browser API explorer, you must enable CORS on your FusionAuth instance. This is not required if you are using the curl commands to make calls against the API from the command line.
```

Comment:
Does the tool require a user to enter their FusionAuth instance domain somewhere? I can't see the Swagger UI.

Reply:
I pulled it down locally. It looks like it uses the servers from `openapi.yaml`. Do users need a custom OpenAPI file in order to hit their actual instance?

Reply:
Yes. See https://github.com/FusionAuth/fusionauth-client-builder/pull/51 where I add in sandbox.fusionauth.io

Otherwise they'll have to run FusionAuth locally. I'll update this doc with that pre-req.